rich/segment.py

```diff
@@ -141,6 +141,7 @@ def _split_cells(cls, segment: "Segment", cut: int) -> Tuple["Segment", "Segment
                     _Segment(before[: pos - 1] + " ", style, control),
                     _Segment(" " + text[pos:], style, control),
                 )
+        raise AssertionError("Can't reach here")
```

Comment:
```suggestion
        raise AssertionError("Can't reach here")  # pragma: no cover
```

Can't see the coverage report, but this is one that could clearly use a no cover pragma.